src/Components/Details.js

```diff
@@ -21,4 +21,4 @@ const Details = ({ data, id }) => (
   </Container>
 );
 
-export default Details;
+export default DetailsComponent;
```

Comment:
`Details` to match declaration

Reply:
Good call! I fixed the `HomePage` one too